### PR TITLE
Make guide and acq detail reports and include links

### PR DESCRIPTION
**Pull request by taldcroft**
This is a start.

To do:
- [x] Put links in to the index.html report

### CODE REVIEW

**Comment by jeanconn** [comment]
Do you want to just cut f13d90c from this PR and I'll finish my reorganization of those checks and the tests for them?

**Comment by jeanconn** [comment]
Oh nevermind, I'm guessing this is based on the other PR.  Fine I'll just do everything as new commits.

**Comment by taldcroft** [comment]
Sorry, should have mentioned this being based on #7.  This will be a trivial rebase once that gets merged.  There was a reason for basing on #7 though I don't recall it now.

**Comment by taldcroft** [comment]
I'm going to propose that the default `--report-level` is `warning`.  That should cover most cases where inspection of the details is helpful.  Thoughts @jeanconn?

**Comment by taldcroft** [comment]
@jeanconn - I've tested this with a local version of 4.3.3 with all the 4.3.3 milestoned proseco PR's merged and the JAN2819P prelim pickle.  It seems to basically work, but I haven't checked on details.  Please have a look with your usual attention to detail.  :smile:

File: aca_preview/preview.py [hunk 1]
```diff
@@ -167,18 +192,90 @@ def get_summary_text(acas):
...
         aca.messages = []  # Warning messages
+        aca.loud = loud
+
+        # Input obsid could be a string repr of a number that might have have
+        # up to 1 decimal point.  This is the case when obsid is taken from the
+        # ORviewer dict of ACATable pickles from prelim review.  Tidy things up
+        # in these cases.
+        if obsid is not None:
+            f_obsid = round(float(obsid), 1)
+            i_obsid = int(obsid)
```
Comment (jeanconn) [inline aca_preview/preview.py]:
```
ine 217, in add_review_methods
    i_obsid = int(obsid)
ValueError: invalid literal for int() with base 10: '20208.1'
```
Also, what's the best way to test entry-point scripts?  I'm used to just having a __name__ == main in the source script to run it from the source dir.

File: aca_preview/preview.py [hunk 2]
```diff
@@ -167,18 +192,90 @@ def get_summary_text(acas):
...
+        :param loud: print processing status info (default=False)
 
         """
         aca.__class__ = cls
         aca.context = {}  # Jinja2 context for output HTML review
         aca.messages = []  # Warning messages
+        aca.loud = loud
+
+        # Input obsid could be a string repr of a number that might have have
+        # up to 1 decimal point.  This is the case when obsid is taken from the
```
Comment (jeanconn) [inline aca_preview/preview.py]: Also, I think this "up to 1 decimal point" may be wrong.  For the examples we've seen, sure that .19999 was a poor representation of .2, but it does not seem impossible to me that we'd have 10 or more splits of a single target in a prelim (2 or 3 seems much more likely I agree but seems not worth breaking over more).

File: aca_preview/preview.py (hunk 1 above)
Comment (jeanconn) [reply aca_preview/preview.py]: I'm guessing you just intended
```
i_obsid = int(f_obsid)
```

File: aca_preview/preview.py (hunk 2 above)
Comment (taldcroft) [reply aca_preview/preview.py]: I set this to 2 decimal points.  I can't immediately find your example of the MATLAB obsid with a poor representation though.

Comment (jeanconn) [reply aca_preview/preview.py]: You are welcome to the early FEB0419 pickle in ASPECT_ICXC/SFE/2019/FEB0419/FEB0419_Test_proseco.pkl .

File: aca_preview/preview.py [hunk 3]
```diff
@@ -47,15 +47,22 @@ def main(sys_args=None):
     parser.add_argument('--outdir',
                         type=str,
                         help='Output directory (default=<load name>')
+    parser.add_argument('--report-level',
+                        type=str,
+                        default='warning',
+                        help="Make reports for messages at/above level "
+                             "('all'|'none'|'info'|'caution'|'warning'|'critical') "
+                             "(default='none')")
```
Comment (jeanconn) [inline aca_preview/preview.py]: Help text out of date with updated default

File: aca_preview/preview.py (hunk 2 above)
Comment (taldcroft) [reply aca_preview/preview.py]: Passes formatting test, and provided inspiration for the next PR.

**Pull request merged**